test/src/main/java/org/apache/accumulo/test/NewTableConfigurationIT.java

```diff
@@ -517,6 +518,23 @@ public void testNtcChaining() throws AccumuloException, AccumuloSecurityExceptio
     }
   }
 
+  @Test
+  public void testIteratorConflictsWithDefault() throws Exception {
+    String[] tableNames = getUniqueNames(2);
+    String table1 = tableNames[0];
+    String table2 = tableNames[1];
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+      // add an iterator with same priority as the default iterator
+      var iterator1 = new IteratorSetting(20, "foo", "foo.bar");
+      assertThrows(IllegalArgumentException.class, () -> client.tableOperations().create(table1,
```

Comment:
Not sure if this test will still pass w/ the changes that checks for exact props w/ different values. If not, may need to relax the test to use exact props or make the check more strict.